src/components/Upload/UploadFile.vue

```diff
@@ -1,30 +1,52 @@
 <template>
-	<div
-		class="card-body flex justify-center border-3 border-gray-400 border-dashed 40vh"
-	>
-		<div class="flex flex-col justify-center items-center">
-			<i class="fa fa-cloud-upload-alt my-8 text-gray-400 text-6xl"></i>
-			<h3 class="my-1 text-md leading-5 text-gray-800">
-				Pretiahnite súbory sem
-			</h3>
-			<p class="my-1 text-md text-gray-600">alebo</p>
-			<div class="mx-auto my-4">
-				<label
-					class="flex text-white rounded bg-primary-500 hover:bg-primary-400 px-4 py-2 cursor-pointer transition duration-500 ease select-none focus:outline-none"
-				>
-					Vybrať súbory
-					<input
-						id="file"
-						ref="file"
-						type="file"
-						multiple
-						class="hidden"
-						@change="handleFileUpload"
-					/>
-				</label>
-				<button v-on:click="submitFile">Submit</button>
+	<div class="space-y-6">
+		<div class="card-body border-3 border-gray-400 border-dashed 40vh">
+			<div class="card-body flex flex-col justify-center bg-primary-100 hover:bg-white">
```

Comment:
This also looks weird - please do not add hover class  to this and add static, white background.

Reply:
Also I think background is light blue, but as it should be drag n drop zone I'd use some different color while hoovering.